apps/system/js/notifications.js

```diff
@@ -84,7 +84,6 @@ var NotificationScreen = {
     // screen
     this.externalNotificationsCount = 0;
     this.unreadNotifications = [];
```

Comment:
One more point regarding re-use of this variable: in original version it's used to show/hide ambient indicator (tiny blue line under the status bar) so this array will be forcibly cleaned in ``function ns_hideNotificationIndicator()``. Also this array is expected to store only 'unread' notifications, i.e. ones which were not seen by user, so it's expected that after, for example, opening list of notifications all of them became read, but continue exist until the message/call/whatever won't be actually opened/read in proper application. I can see here 2 possible solutions:

1. In case if ambient indicator is not expected to be in RedSquare (here we need confirmation for UX team) - completely remove related functionality and re-use ``unreadNotifications``.
2. Otherwise just create new variable for storing notifications (e.g. #21, where similar counters were implemented to be able to show amounts of notifications for individual applications)